### PR TITLE
Apikey search Fixes for admin tool Ui cortellis users

### PR DESCRIPTION
**Pull request by darshan-clvt**
1. This PR is created from apikey_search branch 
2. This PR is for apikey search fixes in admin tool UI 
3. Imported required modules
**NOTE**
Removed aduit log changes to make unique.
This PR to avoid conflicts and quick reverting changes

### CODE REVIEW

File: src/ui/routes/admin.js [hunk 1]
```diff
@@ -178,6 +180,77 @@ function mustBeAdminOrApproverMiddleware(req, res, next) {
...
+            return next(err);
+        }
+        let body = utils.getJson(Apikey.body);
+        if (body && body.message === "Not found") {
+            res.json({
+                title: 'All Subscriptions',
+                subscriptions: null
+            });
+        } else {
+            debug(`i am going inside else `)
```
Comment (saichrn7) [inline src/ui/routes/admin.js]: either remove this debug statement or out some meaningful message

File: src/ui/routes/admin.js [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
 const router = express.Router();
+const request = require('request');
 const async = require('async');
 const mustache = require('mustache');
 const { debug, info, warn, error } = require('portal-env').Logger('portal:admin');
 const tmp = require('tmp');
 const fs = require('fs');
 const util = require('util');
 const utils = require('./utils');
+const process = require('process')
```
Comment (saichrn7) [inline src/ui/routes/admin.js]: is this required ?, if not remove

File: src/ui/routes/admin.js (hunk 1 above)
Comment (darshan-clvt) [reply src/ui/routes/admin.js]: okay

File: src/ui/routes/admin.js (hunk 2 above)
Comment (darshan-clvt) [reply src/ui/routes/admin.js]: will check it

**Review by saichrn7 — APPROVED** [review]

**Pull request merged**